agent/script/parallel_script.go

```diff
@@ -66,6 +66,23 @@ func (s ParallelScript) Run() error {
 	return s.summarizeErrs(passedScripts, failedScripts)
 }
 
+func (s ParallelScript) Cancel() error {
+	s.logger.Debug(s.logTag, "Canceling a parallel script")
+	existingScripts := s.findExistingScripts(s.allScripts)
+	for _, script := range existingScripts {
+		if script, ok := script.(CancellableScript); ok {
```

Comment:
Perhaps, but I like that we intentionally specify that script is cancellable without cluttering all Script implementations. Would be nice to have default implementation in interface like in Java 8.